pkg/elemental/elemental.go

```diff
@@ -410,6 +416,69 @@ func (e *Elemental) DumpSource(target string, imgSrc *v1.ImageSource) (info inte
 		if err != nil {
 			return nil, err
 		}
+	} else if imgSrc.IsOCIFile() {
+		// Extract OCI image from tar file
+		e.config.Logger.Infof("Loading OCI image from tar file %s", imgSrc.Value())
+
+		// Accounting for different image save conventions between tools, load the image from the tar file
+		// First attempt: Try to load without specifying a tag
+		img, err := tarball.ImageFromPath(imgSrc.Value(), nil)
+
+		// Second attempt: If that fails, try with a oci-image:latest tag convention
+		if err != nil {
+			e.config.Logger.Infof("Trying to load with explicit oci-image:latest tag: %v", err)
+			tag, tagErr := name.NewTag("oci-image:latest")
+			if tagErr != nil {
+				e.config.Logger.Errorf("Failed to create tag reference: %v", tagErr)
+				return nil, fmt.Errorf("failed to create tag reference: %w", tagErr)
+			}
+
+			img, err = tarball.ImageFromPath(imgSrc.Value(), &tag)
+			if err != nil {
+				// Third attempt: Try to extract the tar file directly
+				e.config.Logger.Infof("Trying to extract tar file directly: %v", err)
```

Comment:
nit: some of the logging should be debug to not overload the logs with so much stuff